src/LogicalGraph.ts

```diff
@@ -131,13 +131,27 @@ export class LogicalGraph {
 
     static toOJSJsonString(graph : LogicalGraph, forTranslation : boolean) : string {
         let result: string = "";
-
         const json: any = LogicalGraph.toOJSJson(graph, forTranslation);
 
         // NOTE: manually build the JSON so that we can enforce ordering of attributes (modelData first)
         result += "{\n";
         result += '"modelData": ' + JSON.stringify(json.modelData, null, 4) + ",\n";
-        result += '"graphConfigurations": ' + JSON.stringify(json.graphConfigurations, null, 4) + ",\n";
+
+        // if we are sending this graph for translation, then only provide the "active" graph configuration, or an empty array if none exist
+        // otherwise, add all graph configurations
+        if (forTranslation){
+            if (graph.activeGraphConfig().getId() === null){
+                result += '"graphConfigurations": {},\n';
+            } else {
+                const graphConfigurations: any = {};
+                graphConfigurations[graph.activeGraphConfig().getId().toString()] = GraphConfig.toJson(graph.activeGraphConfig());
+
+                result += '"graphConfigurations": ' + JSON.stringify(graphConfigurations, null, 4) + ",\n";
```

Comment:
`const JsonIndent = 4` could be put somewhere, given we use it in a few places.

Reply:
I like this suggestion. I've defined EagleConfig.JSON_INDENT = 4 and used it in this file and others.

Wherever the output JSON might be human-readable, I use this constant to specify the indent width. But wherever the JSON is unlikely to be read by a human, such as when stored in HTML localStorage, or when sent to an EagleServer API, I leave it using the default indent.